JavaSource/org/unitime/timetable/util/InstructionalOfferingRollForward.java

```diff
@@ -297,6 +298,7 @@ private Class_ rollForwardClass(Class_ fromClass,SchedulingSubpart toSubpart, Se
 		toClass.setSchedulePrintNote(fromClass.getSchedulePrintNote());
 		toClass.setSchedulingSubpart(toSubpart);
 		toClass.setUniqueIdRolledForwardFrom(fromClass.getUniqueId());
+		toClass.setFundingDept(fromClass.getFundingDept());
```

Comment:
This does not work. You need to look up the matching funding department in the toSession